lib/plugins/aws/invokeLocal/resolvers/s3Bucket.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+module.exports = {
+  'Fn::GetAtt': (plugin, resource, resourceName, parameters) => {
+    switch (parameters[1]) {
+      case 'Arn':
+        return new Promise(resolve => {
+          resolve(`arn:aws:s3:::${resource.PhysicalResourceId}`);
```

Comment:
There's nothing async in this function, therefore let's remove any promise initializations

Reply:
This is needed to make sure that the resource handlers can have a ".then" action on their result.  Since the version of nodejs you support does not support the async and await declorations, it is not possible to just return a value without causing inconsistency with result handling.

Reply:
>  Since the version of nodejs you support does not support the async and await declorations, it is not possible to just return a value without causing inconsistency with result handling.

That's true, but we can handle it more gentle, and keep resolvers free from noise that affects readabily.

What in async function can be written as:

```javascript
const result = await syncOrAsync();
...
```

Without async/await can be written as:

```javascript
BbPromise.resolve(syncOrAsync(()).then(result => {
  ...
});
```

So my proposal is to use above construct when processing resolvers, and keep resolvers implementation clean (it'll make it more future proof for upcoming async/await support, which we plan to add very shortly)